CLAUDE.md

```diff
@@ -185,6 +197,6 @@ The `proto.yml` workflow runs on pushes to `star-proto/`:
 ## Key Documentation
 
 - `star-esp32-firmware/CLAUDE.md` - Detailed ESP32 firmware guide
+- `star-gateway/CLAUDE.md` - Gateway service architecture and build guide
 - `star-proto/STYLE_GUIDE.md` - Protobuf conventions
```

Comment:
Remote this line and remove that md file if it exists. I'm working on merging to main better docs. I think that this repo should have inside the root dir "docs/"  and inside that is a tex your proto styles. If not leave this as something that we will fix later.